starknet_py/net/models/chains.py

```diff
@@ -27,3 +28,16 @@ def chain_from_network(
         raise ValueError("Chain is required when not using predefined networks.")
 
     return chain
+
+
+def default_token_address_for_chain(chain_id: Optional[StarknetChainId] = None) -> str:
```

Comment:
i struggle to understand why the chain_id can be `None` here, kind of weird function definition.

Reply:
It can be None, because 
1. an account can be created without passing the chain
2. if 1 and the user does not provide chain_id in the get_balance => we do not know the chain

Reply:
Can't `StarknetChainId` have this method?